tests/System.Text.Primitives.Tests/PrimitiveParserUInt32PerfTests.cs

```diff
@@ -279,23 +279,28 @@ private unsafe static void PrimitiveParserByteSpanToUInt32(string text)
             }
         }
 
-        [Benchmark(Skip = "The generic type 'System.Collections.Generic.List`1' was used with an invalid instantiation in assembly 'System.Private.CoreLib")]
-        private unsafe static void PrimitiveParserByteSpanToUInt32_VariableLength()
+        [Benchmark]
+        [InlineData(0)]
```

Comment:
If you look at some of the other tests, they essentially do the same thing:
https://github.com/dotnet/corefxlab/blob/master/tests/System.Text.Primitives.Tests/PrimitiveParserUInt32PerfTests.cs#L466

Instead of passing in index to a string[], they just pass in the strings themselves. I could do that, if you prefer.